### PR TITLE
Redirects users to the login page after an expired session and notify them with a toast

### PR DESCRIPTION
**Pull request by TheoCathy**
…to the login page

**What kind of change does this PR introduce?**
Bug fix

**Issue Number:**
#656 
Fixes #

**Did you add tests for your changes?**
No

**Snapshots/Videos:**

**If relevant, did you update the documentation?**

**Summary**
Users will be notified when the session is expired and redirects back to login page rather than them manually clicking on login button to be able to use the platform again. It also gives the appropriate notification. 

**Does this PR introduce a breaking change?**
No

**Other information**

**Have you read the [contributing guide](https://github.com/PalisadoesFoundation/talawa-admin/blob/master/CONTRIBUTING.md)?**
yes

### CODE REVIEW

File: src/screens/LoginPage/LoginPage.tsx [hunk 1]
```diff
@@ -189,6 +189,8 @@ function LoginPage(): JSX.Element {
         }
       } else {
         toast.warn('User not found!');
+        // toast.warn('User not found! kindly relogin as sessison has expired');
+        // window.location.replace('/login');
```
Comment (noman2002) [inline src/screens/LoginPage/LoginPage.tsx:193]: We already have toasts. Use that.

File: src/components/SecuredRoute/SecuredRoute.tsx [hunk 2]
```diff
@@ -13,4 +13,24 @@ const SecuredRoute = (props: any): JSX.Element => {
...
+  const currentTime = Date.now();
+  const timeSinceLastActive = currentTime - lastActive;
+  const logoutUserTime = 20 * 60 * 1000;
+
+  if (timeSinceLastActive > logoutUserTime) {
+    toast.error(
+      'Session timed out. Please login again to keep using the platform'
+    );
+    window.location.href = '/';
+    localStorage.setItem('IsLoggedIn', 'FALSE');
```
Comment (noman2002) [inline src/components/SecuredRoute/SecuredRoute.tsx:32]: Use the already existing toast.

**Review by noman2002 — CHANGES_REQUESTED** [review]

File: src/components/SecuredRoute/SecuredRoute.tsx (hunk 2 above)
Comment (TheoCathy) [reply src/components/SecuredRoute/SecuredRoute.tsx:32]: So to clarify, are you saying I should use the existing toast but leave the inactive timeout function?

Comment (noman2002) [reply src/components/SecuredRoute/SecuredRoute.tsx:32]: don't leave the function, just use the existing toast.

**Comment by palisadoes** [comment]
@TheoCathy Any updates on the requested changes?

**Comment by TheoCathy** [comment]
@palisadoes Yes all done was merged in #734 PR... we can close this or use it to update the toast message "User not found!" to a more user friendly message like " Invalid credentials, User not found" as earlier pointed out. What do you think?

**Pull request closed without merging**